### ISSUE TITLE
FFMPEG build fail

### ISSUE THREAD

**Issue by tobozo** [andybalaam/retropie-recording#1]
ffmpeg wouldn't compile which is no surprise after three months of commits between the creation of the bash script and now.

so I tried every commit back to December until it worked

it was solved by issuing this command after cloning the repo && cd x264 :

`git checkout 5d4031618e9feedcb527fd4e5a91bc06e30b70b4`

[edit]

also using this release of fdk-aac since it evolved too: https://github.com/mstorsjo/fdk-aac/archive/v0.1.5.tar.gz

**Comment by andybalaam** [comment]
Thanks for finding these - if you get around to a pull request I'll gladly accept it.

**Comment by tobozo** [comment]
I don't have the image of the Pi any more, I moved the microSD from a Pi3B to a Pi3B+ and it messed up the system so badly I had to reinstall from scratch.

I was using an old RetroPie based on wheezy or jessie, can't remember which of those, but it didn't like being loaded on the latest Pi3B+ model.

So I downloaded/installed the latest version (RetroPie 4.4) and compiled ffmpeg without having to checkout an older version; but I also realized it was useless as RetroArch has now a [built in support](https://github.com/libretro/RetroArch/blob/32b2808fe0c7bc5b38b9896a8c2f91022cf7a926/record/drivers/record_ffmpeg.c).

Anyway, before I wrecked my installation I had the time to do a capture of [Zerowing](https://youtu.be/dgGZP87G2uA) and this was my initial motivation.

Thanks a lot for providing this project!
